commons/utils/s3_utils.py

```diff
@@ -379,9 +379,10 @@ def create_multipart_json(json_path, parts_list) -> tuple:
 
     return os.path.exists(json_path), json_path
 
+
 def assert_s3_err_msg(rgw_error, cortx_error, cmn_cfg, error):
     """Checks the s3 engine type and asserts accordingly """
     if const.S3_ENGINE_RGW == cmn_cfg:
-        assert_utils.assert_equal(rgw_error, error.message, error.message)
+        assert_utils.assert_in(rgw_error, error.message, error.message)
```

Comment:
Replaced  assert_equals with assert_in for the following 2 reasons.
1. Most of our automation code is checking specific Error Msg key word instead of matching whole  error msg.
This reduces length of error_message.py constant file.
2) Also, it has been found that 2 or more error msg might have similar Key word like (InvalidTagError) but rest of msg is different. Instead of checking full msg, checking subset of string will help to reduce space.
(for example, PutBucketTagging and PutObjectTagging both have **InvalidTag** keyword, but whole msg will differ since msg includes putbucket/object operation names)